### PR TITLE
Update provision README

### PR DESCRIPTION
**Pull request by jskinne3**
My attempts to:
1. correct / clarify variable names and meanings
2. record my struggle and its solution

I would love more knowledgable people to pick this over

### CODE REVIEW

File: terraform/provision/README.md [hunk 1]
```diff
@@ -27,11 +30,25 @@ the broker context here.
...
+## Troubleshooting
+
+```
+Error creating SES domain identity verification: Expected domain verification Success, but was in state Failed
+```
+This error occurs when [the timeout](https://github.com/GSA-TTS/datagov-brokerpak-smtp/blob/767bcb71179494a0578c018f8338df4711f1c4fc/terraform/provision/verification.tf#L61) to verify the domain identity is reached. This can be a DNS problem.
+
+Ensure that the domain or subdomain you are working in is reachable by DNS. Ensure that the zone(s) above your subdomain have DNS records which point to your zone &mdash; for example if you are using a `default_domain` of `dev.ssb.notify.gov`, ensure that the DNS records at `ssb.notify.gov` correctly indicate a `dev` subdomain. It will need an NS and a DS record.
+
+The [outputs.tf file of usnotify-ssb](https://github.com/GSA/usnotify-ssb/blob/main/outputs.tf#L16) generates instructions on creating DNS records.
```
Comment (mogul) [inline terraform/provision/README.md:54]: This should refer to the current repository.

**Review by mogul — CHANGES_REQUESTED** [review]
Other than this, it looks good. I'm glad you were able to figure it out!

File: terraform/provision/README.md (hunk 1 above)
Comment (jskinne3) [reply terraform/provision/README.md]: I have removed the reference to this Notify.gov-specific repo

**Review by mogul — APPROVED** [review]
I'm fine with removing the line, but I meant just changing the text to omit mention of Notify, and have the URL report to the current repo's `outputs.tf`! Merge if you want, or make that change.

**Comment by jskinne3** [comment]
OK. In my case it was the SSB output that contained the DNS records that fixed the problem I'm reporting on here. So I've included instructions on getting output in both locations. I eliminated the reference to Notify by pointing to the original SSB rather than Notify's fork.

**Review by mogul — APPROVED** [review]

**Pull request merged**